### DISCUSSION TITLE
Permissions diagnosis

### DISCUSSION THREAD

**Discussion by casey-chow** [directus/directus#5794] [Archive]
Is there a way to know what permissions are missing in the app when a user is deemed not to have sufficient privileges? I have users who are getting hit with a lot of errors and I would like to be able to use the failures to tune my ACLs.

**Comment by benhaynes** [comment]
Under permissions, you can click "System Collections" and then "App Access Minimum" or "Recommended Defaults" to reset them. This could also be used in a dummy role for comparison.

![Screen Shot 2021-05-20 at 5 52 05 PM](https://user-images.githubusercontent.com/522079/119053481-1c897180-b994-11eb-9fb7-42100b4ca5f4.png)

**Reply by casey-chow** [reply]
The failures seem to be on application-level fields actually. The main issue I'm having is knowing what permissions to set for dependent collections--for instance, I didn't realize that a user needed access to modify my `created_at` field on update, and re-enabling modifying that field fixed permissions errors for one particular changee.
